src/OpenTelemetry.Exporter.Console/OpenTelemetry.Exporter.Console.csproj

```diff
@@ -30,13 +30,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="$(RepoRoot)\src\Shared\Guard.cs" Link="Includes\Guard.cs" />
+    <Compile Include="$(RepoRoot)\src\Shared\Guard.cs" Link="Includes\Guard.cs" Condition="'$(ExposeExperimentalFeatures)' == 'true'" />
```

Comment:
nit: Create a new `ItemGroup` with this condition and place the three shared files under it.

Reply:
Updated